test/powershell/Host/TabCompletion/BugFix.Tests.ps1

```diff
@@ -35,4 +35,18 @@ Describe "Tab completion bug fix" -Tags "CI" {
         $result.CompletionMatches[1].CompletionText | Should Be "-NoClobber"
         $result.CompletionMatches[2].CompletionText | Should Be "-NoOverwrite"
     }
+    It "Issue#3416 - 'Select-Object -ExcludeProperty <tab>' should work" {
```

Comment:
I already have this issue opened to eventually address this: https://github.com/PowerShell/PowerShell/issues/3369